Files kdebase-3.5.5/kicker/data/kickoff-icons.tar.bz2 and kicker-kickoff/data/kickoff-icons.tar.bz2 differ
diff -Nurp kdebase-3.5.5/kicker/kicker/buttons/knewbutton.cpp kicker-kickoff/kicker/buttons/knewbutton.cpp
--- kdebase-3.5.5/kicker/kicker/buttons/knewbutton.cpp	2006-10-30 01:24:22.000000000 +0100
+++ kicker-kickoff/kicker/buttons/knewbutton.cpp	2006-11-03 15:17:48.000000000 +0100
@@ -122,6 +122,7 @@ bool KNewButton::eventFilter(QObject *o,
         }
     }
 
+
     if (e->type() == QEvent::MouseMove)
     {
         QMouseEvent *me = static_cast<QMouseEvent *>(e);
@@ -132,6 +133,8 @@ bool KNewButton::eventFilter(QObject *o,
     return KButton::eventFilter(o, e);
 }
 
+
+
 void KNewButton::drawEye()
 {
 #define eye_x 62
@@ -153,12 +156,12 @@ void KNewButton::drawEye()
         if(!m_movie.isNull() && m_movie.running() && !m_movie.framePixmap().isNull())
             m_pixmap = m_movie.framePixmap();
 
-        QPainter p(&m_pixmap);
+        /* QPainter p(&m_pixmap);
         p.setPen(white);
         p.setBrush(white);
         //      p.setPen(QColor(110,185,55));
         p.drawRect(eye_x+dx, eye_y+dy, 2, 2);
-        p.end();
+        p.end(); */
 
         repaint(false);
     }
@@ -166,6 +169,8 @@ void KNewButton::drawEye()
 #undef eye_y
 }
 
+
+
 void KNewButton::enterEvent(QEvent* e)
 {
      KButton::enterEvent(e);
diff -Nurp kdebase-3.5.5/kicker/kicker/ui/itemview.cpp kicker-kickoff/kicker/ui/itemview.cpp
--- kdebase-3.5.5/kicker/kicker/ui/itemview.cpp	2006-10-30 01:24:21.000000000 +0100
+++ kicker-kickoff/kicker/ui/itemview.cpp	2006-11-03 11:42:57.000000000 +0100
@@ -148,7 +148,7 @@ void KMenuItem::setup()
 {
     // if someone configured a larger generalFont than 10pt, he might have a _real_ problem with 7pt
     // the 7pt could be read out of konquerorrc I guess
-    float min_font_size = 7. * QMAX(1., KGlobalSettings::generalFont().pointSizeFloat() / 10.);
+    float min_font_size = 10. * QMAX(1., KGlobalSettings::generalFont().pointSizeFloat() / 10.);
 
     const int expected_height = 38;
     description_font_size = QMAX( pointSize( expected_height * .3, listView() ), min_font_size );
@@ -355,7 +355,7 @@ void KMenuItemSeparator::setup()
 
     QFont f = QFont();
     QFontMetrics fm(f);
-    f.setPointSize( 8 );
+    f.setPointSize( 10 );
     if ( itemAbove() && !text( 0 ).isEmpty() )
         setHeight( (int)QMAX( 34., fm.height() * 1.4) );
     else
@@ -407,7 +407,7 @@ void KMenuItemSeparator::paintCell(QPain
       }
       QFont f = listView()->font();
       f.setWeight( QFont::Normal );
-      f.setPointSize( 8 );
+      f.setPointSize( 10 );
       p->setFont( f );
       QColor myColor = cg.color( QColorGroup::Text ).light( 200 );
       if ( qGray( myColor.rgb() ) == 0 )
diff -Nurp kdebase-3.5.5/kicker/kicker/ui/k_new_mnu.cpp kicker-kickoff/kicker/ui/k_new_mnu.cpp
--- kdebase-3.5.5/kicker/kicker/ui/k_new_mnu.cpp	2006-11-01 18:08:45.000000000 +0100
+++ kicker-kickoff/kicker/ui/k_new_mnu.cpp	2006-11-03 17:04:52.000000000 +0100
@@ -171,7 +171,7 @@ KMenu::KMenu()
     if (!gethostname( hostname, sizeof(hostname) ))
       hostname[sizeof(hostname)-1] = '\0';
 
-    m_userInfo->setText( i18n( "User&nbsp;<b>%1</b>&nbsp;on&nbsp;<b>%2</b>" )
+    m_userInfo->setText( i18n( "Hen&nbsp;<b>%1</b>&nbsp;inside&nbsp;<b>%2</b>&nbsp;farm" )
                          .arg( user->loginName() ).arg( hostname ) );
     setupUi();
 
@@ -364,7 +364,7 @@ KMenu::KMenu()
     pixmapLabel_exit->setFixedSize( tab_icon_size, tab_icon_size );
 
     QFont f = font();
-    f.setPointSize( QMAX( 7, qRound( f.pointSize() * 0.80 ) ) );
+    f.setPointSize( QMAX( 10, qRound( f.pointSize() * 0.80 ) ) );
     m_exit_text->setFont( f );
     m_btnFavorites_text->setFont( f );
     m_btnRecently_text->setFont( f );
@@ -373,7 +373,7 @@ KMenu::KMenu()
     m_system_text->setFont( f );
     m_exit_text->setFont( f );
 
-    f.setPointSize( QMAX( 7, qRound( f.pointSize() * 1.50 ) ) );
+    f.setPointSize( QMAX( 10, qRound( f.pointSize() * 1.50 ) ) );
     m_searchLabel->setFont( f );
 
     int minwidth = 32;
@@ -1441,7 +1441,11 @@ void KMenu::initialize()
     else
         m_lock->hide();
 */
-    connect( m_lock, SIGNAL(clicked()), SLOT(slotOpenSUSE()));
+
+/*    removing OpenSuse branding */
+/*    connect( m_lock, SIGNAL(clicked()), SLOT(slotOpenSUSE())); */
+      m_lock->hide();
+
     if (!kapp->authorize("logout"))
         m_exit->hide();
 
@@ -1580,18 +1584,18 @@ void KMenu::insertStaticItems()
 
     m_systemView->insertSeparator( nId++, i18n("Applications"), index++);
 
-    m_systemView->insertItem( "yast", i18n( "System Settings" ),
-                                     i18n( "YaST2" ), "/usr/share/applications/YaST.desktop", nId++, index++ );
+    m_systemView->insertItem( "kcontrol", i18n( "KDE Settings" ),
+                                     i18n( "KDE Control Center" ), "/usr/kde/3.5/share/applications/kde/KControl.desktop", nId++, index++ );
 
-    m_systemView->insertItem( "info", i18n( "System Information" ),
-                              "sysinfo:/",  "sysinfo:/", nId++, index++ );
+    m_systemView->insertItem( "kcmprocessor", i18n( "System Information" ),
+                              "Information Center",  "/usr/kde/3.5/share/applications/kde/kinfocenter.desktop", nId++, index++ );
 
     m_systemView->insertSeparator( nId++, i18n("System Folders"), index++ );
 
-    m_systemView->insertItem( "folder_red", i18n( "Home Folder" ),
+    m_systemView->insertItem( "folder_home", i18n( "My Documents" ),
                               QDir::homeDirPath(), "file://"+QDir::homeDirPath(), nId++, index++ );
 
-    if ( KStandardDirs::exists( KGlobalSettings::documentPath() + "/" ) )
+/*    if ( KStandardDirs::exists( KGlobalSettings::documentPath() + "/" ) )
     {
         QString documentPath = KGlobalSettings::documentPath();
         if ( documentPath.endsWith( "/" ) )
@@ -1600,6 +1604,8 @@ void KMenu::insertStaticItems()
            m_systemView->insertItem( "folder_green", i18n( "My Documents" ), documentPath, documentPath, nId++, index++ );
     }
 
+*/
+
     m_systemView->insertItem( "remote", i18n( "Network Folders" ),
                               "remote:/", "remote:/", nId++, index++ );
 
@@ -1642,7 +1648,7 @@ void KMenu::slotLock()
 
 void KMenu::slotOpenSUSE()
 {
-    kapp->invokeBrowser("http://opensuse.org");
+    kapp->invokeBrowser("http://www.sabayonlinux.org");
 }
 
 void KMenu::slotLogout()
